easypost/report.py

```diff
@@ -8,7 +8,20 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         """Create a report."""
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        url = "%s/%s" % (cls.class_url(), params["type"]) + "?"
+
+        if "columns" in params:
+            for column in params.get("columns", []):
+                url += f"columns[]={str(column)}&"
+            # Delete the columns from the param sinces it's added in the url query.
+            params.pop("columns")
+
+        if "additional_columns" in params:
+            for column in params.get("additional_columns", []):
+                url += f"additional_columns[]={str(column)}&"
```

Comment:
I'm unsure that we need to wrap this in str() since we expect them to be strings already. If they passed in say integers, then coercing it to a string won't do any good because it will still fail at the API level.

Reply:
you're correct, this is redundant